src/StreamsHandler.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include "src/Common.h"
+#include "src/Payload.h"
+
+namespace reactivesocket {
+
+class FrameSerializer;
+
+class StreamsWriter {
+ public:
+  virtual ~StreamsWriter() = default;
+  virtual void writeNewStream(
+      StreamId streamId,
+      StreamType streamType,
+      uint32_t initialRequestN,
+      Payload payload,
+      bool TEMP_completed) = 0;
```

Comment:
this is just because the parameter is temporary and will be remove in the next 2 diffs.
I wanted to make it visible.